app/extend/context.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
 
   get nonce() {
     if (!this._nonceCache) {
-      this._nonceCache = rndm(16);
+      this._nonceCache = nanoid(16);
```

Comment:
according benchmark result https://github.com/ai/nanoid#benchmark
nanoid will 10x slow then rndm(non-secure) version.

Reply:
Yeap, this is a price for security. You can use `nanoid/non-secure`, but I think having real random numbers is more important than microoptimization here 😊

Reply:
@ai：If there's no obvious secure problem. I don't think it needs modifications, for 10x slower.

Reply:
@Maledong good point :+1: If you want only performance I replaced `nanoid` with `nanoid/non-secure`, which is 2-3% faster than `rndm`. But I will understand if 2-3% of performance is not a big deal

(Of course, you can also remember that `nanoid` is a more active project and have more test, including statistic distribution tests, but it is not big deal as well).

Reply:
Decision on @fengmk2 :D